CONTRIBUTING.adoc

```diff
@@ -13,11 +13,12 @@ Before you submit a new issue, we recommend that you search the list of issues t
 
 Thank you for your contributions! Please follow this process to submit a patch:
 
-.  Create an issue describing your proposed change to the repository.
-.  The repository owners will triage and respond to your issue promptly.
-.  Fork the repository and create a topic branch.
-.  Refer to the link:https://docs.openshift.org/latest/minishift/contributing/developing.html[developer documentation] for guidelines and tips on how to build and test Minishift.
-.  Submit a pull request with the proposed changes.
+. Create an issue describing your proposed change to the repository.
+. The repository owners will triage and respond to your issue promptly.
+. Fork the repository and create a topic branch.
+. Refer to the link:https://docs.openshift.org/latest/minishift/contributing/developing.html[developer documentation] for guidelines and tips on how to build and test Minishift.
+. Submit a pull request with the proposed changes.
+  To make reviewing and integration as smooth as possible, please run the targets link:https://docs.openshift.org/latest/minishift/contributing/developing.html#unit-tests[`test`], https://docs.openshift.org/latest/minishift/contributing/developing.html#format-source-code[`fmtcheck`] and https://docs.openshift.org/latest/minishift/contributing/developing.html#commit-messages[`validate_commits`] prior to submitting the pull request.
```

Comment:
Should we clarify that these are `make` targets, or is the context enough to convey this information?

Reply:
Will clarify